pyorbital/tests/test_orbital.py

```diff
@@ -298,6 +298,12 @@ def _xarr_conv(input):
         np.testing.assert_allclose(azi.data.compute(), self.exp_azi)
         np.testing.assert_allclose(elev.data.compute(), self.exp_elev)
 
+    def test_scalar(self):
+        """Test with scalar inputs."""
+        from pyorbital.orbital import get_observer_look
+        (azi, elev) = get_observer_look(0, 0, 30_000_000, self.t, 0, 0, 0)
```

Comment:
Huh?  Am I crazy or is stickler crazy?  This syntax is fine...

Reply:
Maybe stickler doesn't understand your `_` separators. If you run flake8 on your local file does it pass? Otherwise I would also prefer you remove the unnecessary `()` around the assignment. :wink:

Reply:
If I run flake8 locally it passes.  The `_`-separators have been valid Python syntax since 3.0 and I think they add clarity here.

As for the parenthesis, do we have a style guide for this issue?  I'm used to always including the optional parenthesis around tuples.

Reply:
See also the discussion in the Pytroll Slack from https://pytroll.slack.com/archives/C023KKJ7QRM/p1639556296083600 onwards.

Reply:
Stickler was using Python 2.  On the parenthesis, it seems the consensus from the slack discussion is to allow both, so I'm going to leave them as is.